wayland/xorgless.txt

```diff
@@ -0,0 +1,64 @@
+Xorgless
+________________________________________________________________________________
+
+Running Wayland without any software depending on X11 is very well supported
+nowadays, with many popular X packages (such as dmenu, st, xclip) having
+completely Wayland-only analogues!
+
+To build a Wayland-only repository quickly, you can clone $/dilyn-corner/KISS-me
+
+There are only two packages from repo/xorg you will need: libxkbcommon and
+xkeyboard-config.
+
+Fork the packages you'll need to modify:
+
+for pkg in                              \
+    libxkbcommon xkeyboard-config libva \
+    mesa freetype-harfbuzz cairo pango  \
+    gdk-pixbuf gtk+3 mpv firefox        \
+do
+    kiss-fork $pkg
+done
+
+Modify the depends file for each of these packages to remove any '*X*' packages.
+
+Most packages should be to check for X11 and Wayland support at build-time
+without having to specify any specific options. However, you will have to modify
+some; change the following build files:
+
++---------------+--------------------------------------------------------------+
+| Package       | Option                                                       |
++---------------+--------------------------------------------------------------+
+| mesa          | Switch 'x11' to 'wayland' in -Dplatforms                     |
```

Comment:
Mesa also needs to specify `-Dglx=disabled` otherwise its expecting libX11

Reply:
Very true! I had forgotten.